cupy/manipulation/rearrange.py

```diff
@@ -1,6 +1,33 @@
 import cupy
 
 
+def flip(a, axis):
+    """Reverse the order of elements in an array along the given axis.
+
+    Args:
+        a (~cupy.ndarray): Input array.
+        axis (int): Axis in array, which entries are reversed.
+
+    Returns:
+        ~cupy.ndarray: Output array.
+
+    .. seealso:: :func:`numpy.flip`
+
+    """
```

Comment:
It would be better to describe remarks that this document is just copied or altered from the original one.
See: https://github.com/pfnet/chainer/blob/master/docs/source/contribution.rst#coding-guidelines